src/formatDate.js

```diff
@@ -51,6 +51,52 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const arr = [];
+  const arrSplit = date.split(fromFormat.pop());
+  const arrToFormat = toFormat.pop();
+  let year = '',
+    month = '',
+    day = '';
+
+  for (let idx = 0; idx < fromFormat.length; idx++) {
```

Comment:
`idx` is bit weird, common iterators are `i / j`
although that's not an error, yet better get used to common conventions